db/db.go

```diff
@@ -50,8 +51,8 @@ var maxSqlNumber = uint64(9223372036854775807)
 
 const MaxSqlInteger = 2147483647
 
-var addressLikeRE = regexp.MustCompile(`^(0x)?[0-9a-fA-F]{0,40}$`)
-var blsLikeRE = regexp.MustCompile(`^(0x)?[0-9a-fA-F]{0,96}$`)
+var addressRE = regexp.MustCompile(`^(0x)?[0-9a-fA-F]{40}$`)
+var blsRE = regexp.MustCompile(`^(0x)?[0-9a-fA-F]{96}$`)
```

Comment:
**suggestion:** We already have the **addressRE** as **eth1AddressRE** in **utils**.
We could move the **blsRE** over there to have the regular expression more grouped together.

Reply:
would rather let it be like it is, we can change this in another pr